src/components/Kv/KvLightbox.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+	<transition name="fade">
+		<div v-show="isShown" class="kv-lightbox">
+			<div class="row">
+				<div class="columns small=12">
+					<div class="lightbox-content" :class="removeContentBg">
+						<a @click.stop.prevent="closeLightbox" class="close-lightbox" aria-label="Close">
```

Comment:
this should be a `<button>` since it's a clickable that isn't navigating anywhere.